lib/v2/cib/whpj.js

```diff
@@ -0,0 +1,78 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const https = require('https');
+const crypto = require('crypto');
+
+module.exports = async (ctx) => {
+    // fix unsafe legacy renegotiation disabled
+    const agent = new https.Agent({
+        secureOptions: crypto.constants.SSL_OP_LEGACY_SERVER_CONNECT,
+    });
+
+    const response = await got('https://personalbank.cib.com.cn/pers/main/pubinfo/ifxQuotationQuery.do', { agent: { https: agent } });
+    const cookies = response.headers['set-cookie'].map((item) => item.split(';')[0]).join(';');
+
+    const $ = cheerio.load(response.data);
+    let date = $('div.main-body').find('div.labe_text').text();
+    date = date.split('\n\t')[1].replace('日期：', '').trim();
+    date = date.substring(0, 11) + date.substring(15);
+
+    const link = 'https://personalbank.cib.com.cn/pers/main/pubinfo/ifxQuotationQuery/list?_search=false&dataSet.rows=80&dataSet.page=1&dataSet.sidx=&dataSet.sord=asc';
+    const data = await ctx.cache.tryGet(link, async () => {
```

Comment:
Please set the 4th parameter of `tryGet` to `false`. Otherwise, the cache will continue to extend its expiry and it won't receive any updates.

docs: https://docs.rsshub.app/joinus/advanced/use-cache